config/default.yml

```diff
@@ -29,6 +29,10 @@ RSpec/EmptyExampleGroup:
   Enabled: true
   CustomIncludeMethods: []
 
+RSpec/ExpectActual:
+  Description: 'Checks for `expect(...)` with literal values'
```

Comment:
I'm just going to change it to

> Checks for `expect(...)` calls containing literal values

I don't think "left hand side" makes things very clear